test/integ/test_arrow_result.py

```diff
@@ -53,8 +68,8 @@
     "INTEGER": [
         None,
         0,
-        99999999999999999999999999999999999999,
-        -99999999999999999999999999999999999999,
+        999999999,
+        -999999999,
     ],
```

Comment:
I'm curious why we changed the test value here

Reply:
Iceberg does not support such large numbers in integer types. I could add another exception for this case if you think it's important for the base test.

Reply:
yeah my concern was we lack coverage for large ints, but as long as it's covered by other tests we should be good